src/sentry/replays/usecases/query/configs/scalar.py

```diff
@@ -43,8 +47,21 @@
     "user.id": StringColumnField("user_id", parse_str, StringScalar),
     "user.ip_address": StringColumnField("ip_address_v4", parse_str, IPv4Scalar),
     "user.username": StringColumnField("user_name", parse_str, StringScalar),
+    "new_error_ids": ComputedField(parse_uuid, ErrorIdScalar),
+    "warning_ids": StringColumnField("warning_id", lambda x: str(parse_uuid(x)), StringScalar),
+    "info_ids": ComputedField(parse_uuid, InfoIdScalar),
```

Comment:
These fields can't exist here.  We assume that for varying columns the row is shared with static information like browser.  Because these rows came to us "out-of-band" they won't have any of that information.  We'll need to search these using the aggregate config.